docs/content/en/docs/reference/artifacts.md

```diff
@@ -305,60 +305,57 @@ These steps use `image-builder` to create an Ubuntu-based or RHEL-based image fo
    ```
 1. Get `image-builder`:
     ```bash
-    cd /tmp
-    sudo wget https://anywhere-assets.eks.amazonaws.com/releases/bundles/21/artifacts/image-builder/0.1.2/image-builder-linux-amd64.tar.gz
-    sudo tar xvf image-builder*.tar.gz
-    sudo cp image-builder /usr/local/bin
+    curl -#o- https://anywhere-assets.eks.amazonaws.com/releases/bundles/21/artifacts/image-builder/0.1.2/image-builder-linux-amd64.tar.gz | \
+      sudo tar -xzC /usr/local/bin ./image-builder
     ```
-1. Create a Bare Metal configuration file (for example, `baremetal.json`) to identify the location of a Red Hat Enterprise Linux 8 ISO image and related checksum and Red Hat subscription information:
+
+1. Create an Ubuntu or Red Hat image.
+
+   **Ubuntu**
+
+   Run `image-builder` with the following options:
+
+      * `--os`: `ubuntu`
+      * `--hypervisor`: For Bare Metal use `baremetal`
+      * `--release-channel`: Supported EKS Distro releases include 1-20, 1-21, 1-22, 1-23, and 1-24.
```

Comment:
Are we supporting 1.20 for EKS-A v0.13? Our support doc notes Nov 1, 2022 as end of support for 1.20: https://anywhere.eks.amazonaws.com/docs/reference/support/support-versions/

Reply:
Perhaps we should generalize this help comment instead of specifying "supported versions".

How does this sound:

```
A supported EKS Distro release in the form '<major>-<minor>' (example: 1-24); see [support-versions](https://anywhere.eks.amazonaws.com/docs/reference/support/support-versions/).
```

Reply:
Yes. Very good.